test/testenv/verificationutils.go

```diff
@@ -375,3 +375,15 @@ func VerifyIndexerClusterScalingUp(deployment *Deployment, testenvInstance *Test
 		return idxc.Status.Phase
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(splcommon.PhaseScalingUp))
 }
+
+// VerifySecretObjectUpdate Check whether the secret object info is pushed to other pods
+func VerifySecretObjectUpdate(deployment *Deployment, testenvInstance *TestEnv, verificationPods []string, secretKey string, modifiedKey string) bool {
+	for _, pod := range verificationPods {
+		key := GetSecretKey(deployment, testenvInstance.GetName(), secretKey, pod)
+		if key != modifiedKey {
+			testenvInstance.Log.Info("Key not updated on pod", "Standalone Pod", pod)
```

Comment:
I guess the "standalone pod" here is nothing to do with the Standalone deployment?

Reply:
typo. Generic function. Should be just pod

Reply:
updated